data/json/items/comestibles.json

```diff
@@ -2624,9 +2646,30 @@
     "material": "hflesh",
     "volume": 2,
     "charges": 6,
-    "flags": [ "EATEN_HOT" ],
+    "flags": [ "EATEN_HOT", "SMOKED" ],
     "fun": 5
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "mannwurst_raw",
+    "name": "raw Mannwurst",
+    "weight": 220,
+    "color": "red",
+    "spoils_in": 24,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -1,
+    "healthy": -1,
+    "nutrition": 32,
+    "description": "A hefty raw long pork sausage that has been prepared for smoking.",
```

Comment:
pork?

Reply:
[long pork](https://en.wiktionary.org/wiki/long_pork)

Reply:
We should probably add notes in there so that we don't have to keep explaining that old euphemism to people.

Reply:
I hope writing it down as 'long-pork' will help to recognize that it's something 'special'. I don't deny, I also thought it was either real pork or something along 'pulled pork' the first time I saw it. I'll add it to nearest commit.

Reply:
Considering it uses as ingredient human flesh, I don't think it really needs to be specifically noted, no? That said, surprised that it's not common knowledge.

Reply:
> That said, surprised that it's not common knowledge.

Not everyone here is a native English speaker.